3_bidirectional_cxx_interop/cmake/modules/AddSwift.cmake

```
@@ -61,6 +66,8 @@ function(_swift_generate_cxx_header target header)
       -module-name "${ARG_MODULE_NAME}"
       -cxx-interoperability-mode=default
       -emit-clang-header-path ${header_path}
+      -emit-dependencies
+    DEPFILE ${depfile_path}
```

We should quote this to ensure that the project will work if folks have a space in their file path.

```suggestion
    DEPFILE "${depfile_path}"
```